src/calculateRentalCost.js

```diff
@@ -5,6 +5,23 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const PAY_PER_DAY = 40;
+  const BIG_DISCOUNT = 50;
+  const SMALL_DISCOUNT = 20;
+  const SHORT_TERM = 3;
+  const LONG_TERM = 7;
+
+  if (days < SHORT_TERM) {
+    return days * PAY_PER_DAY;
+  }
+
+  if (days >= LONG_TERM) {
+    return days * PAY_PER_DAY - BIG_DISCOUNT;
+  }
+
+  if (days >= SHORT_TERM) {
+    return days * PAY_PER_DAY - SMALL_DISCOUNT;
```

Comment:
The last `if` condition is unnecessary because if the execution reaches this point, it means that `days` is already greater than or equal to `SHORT_TERM` and less than `LONG_TERM`. You can remove this `if` statement and directly return the calculation for the small discount.